examples/analysis/pubspec.yaml

```diff
@@ -7,3 +7,7 @@ environment:
 dev_dependencies:
   examples_util: {path: ../util}
   pedantic: ^1.0.0
+
+dependency_overrides:
+  analyzer: 0.39.1
```

Comment:
Issue fixed it seems -- see https://github.com/dart-lang/sdk/issues/39702#issuecomment-563028914 -- so it should be possible to drop these overrides now.

Reply:
Thanks!